ONSdigital/eq-author
# 416 new sidebar

### What is the context of this PR?

Implements new sidebar design:

- updated colours
- moved add page/section into popout menu
- pages now added below the currently selected page
- consolidated page/section links into one component

### How to review 

- check against designs
- check can add page
- check page is added in expected place (below selected page)
- check pages can still be moved/deleted as expected
- check keyboard accessibility
